napari/layers/_markers_layer/_constants.py

```diff
@@ -0,0 +1,31 @@
+from enum import Enum
+
+
+class Symbols(Enum):
+    """Symbols: Valid symbol/marker types for the Markers layer.
+    The string method returns the valid vispy string.
+
+    See the Vispy documentation for more details
+    on the MarkersVisuals:
+        http://vispy.org/visuals.html#vispy.visuals.MarkersVisual
+    """
+    ARROW = 0
```

Comment:
IMO you should be using strings instead of integers but it's not a big deal

Reply:
I went back and forth on strings vs. integers. Ultimately, I settled on integers because it makes it easy to get the index for updating the symbol type drop down. If you  or anybody has a strong preference, I am happy to change it.

Reply:
Yeah, I saw that later in the code. Definitely makes sense 👍